### ISSUE TITLE
question

### ISSUE THREAD

**Issue by duans0217** [tihmstar/futurerestore#161]
i hava a question to the nonceEnabler.  my devide ipad pro wifi , i was  jailbreak  on ios 10.2 with yalu102 b7  and write the generate to the nvram through a valid 10.2 shsh2 blogs (nvram com.apple.System.boot-nonce=0x62e1e2495d654857)， now my device is upgrade to 10.3.3 through shift + restore method , so can i still use the generate and shsh2 to restore 10.2 ?  does the generate value  be removed when i  upgrade to 10.3.3 ? 
thanks!

**Comment by jihyo-song** [comment]
Yes, the generator value in nvram will be removed / cleared when you do update / upgrade.

**Comment by duans0217** [comment]
thanks! one more question ,  does the generator value in nvram will be removed when i erase throuth icloud on https://www.icloud.com ?  what's the difference between  shift + restore and  icloud erase ?

**Comment by duans0217** [comment]
i try it yesterday ,  and success to restore to 10.2 with option --no-baseband , so does that mean the generator value in nvram would not  be remove  when erase throuth icloud on https://www.icloud.com ?
